src/main/java/org/truffleruby/core/exception/CoreExceptions.java

```diff
@@ -591,68 +600,58 @@ public DynamicObject nameError(String message, Object receiver, String name, Nod
     public DynamicObject noMethodError(String message, Object receiver, String name, Object[] args, Node currentNode) {
```

Comment:
Is this still called?
Ideally all NoMethodError would compute the message lazily, right?

Reply:
Seems to still be called via `noMethodErrorUnknownIdentifier` which is used in a couple of places.  Since the exception message is already known generated by the `UnknownIdentifierException` exception I'm not sure there's any value in refactoring it.